docs/source/user_guide_kg_builder.rst

```diff
@@ -426,3 +425,43 @@ It is possible to create a custom writer using the `KGWriter` interface:
 
 
 See :ref:`kgwritermodel` and :ref:`kgwriter` in API reference.
+
+
+Entity Resolver
+===============
+
+The KG Writer component creates new nodes for each identified entity
+without making assumptions about entity similarity. The Entity Resolver
+is responsible for refining the created knowledge graph by merging entity
+nodes that represent the same real-world object.
+
+In practice, this package implements a single resolver that merges nodes
+with the same label and identical "name" property.
+
+.. warning::
+
+    The `SinglePropertyExactMatchResolver` **replaces** the nodes created by the KG writer.
+
+
+It can be used like this:
+
+.. code:: python
+    from neo4j_graphrag.experimental.components.resolver import (
+        SinglePropertyExactMatchResolver,
+    )
+    resolver = SinglePropertyExactMatchResolver(driver)
+    res = await resolver.run()
+
+.. warning::
+
+    By default, all nodes with the __Entity__ label will be resolved.
+    To exclude specific nodes, a filter_query can be added to the query.
+    For example, if a `:Resolved` label has been applied to already resolved entities
+    in the graph, these entities can be excluded with the following approach:
+
+    .. code:: python
```

Comment:
Does this get rendered correctly due to the indentation?

Reply:
Good catch, missing a blank line

Reply:
https://github.com/neo4j/neo4j-graphrag-python/pull/160/commits/2aab66e2bbebbb76f3d48bccee9792e80573c213